templates/bin/run

```diff
@@ -1,4 +1,3 @@
 #!/bin/sh
 
-(sleep 4 && open 'http://[project-name].localhost:3000') &
```

Comment:
why the sleep?

Reply:
Finally! I found this very disturbing. L introduced this for comfort. The sleep should give the browser time to open.

Reply:
@coorasse @schmijos I guess the sleep would not give the browser time to open (becase it gets opened after 4s), but to the server to spin up. see it the same as you guys. everyone can add it to his/her personal setup if wanted (e.g. hotel-app)